### PR TITLE
Set up application structure for Admin Client

### PR DESCRIPTION
**Pull request by ctl-lab**
[PT](https://www.pivotaltracker.com/story/show/175137671)

## This PR contains
- Scaffolding of React Application
- Clean up unnecessary files and comments.

### CODE REVIEW

File: public/manifest.json [hunk 1]
```diff
@@ -0,0 +1,25 @@
+{
+  "short_name": "React App",
+  "name": "Create React App Sample",
+  "icons": [
```
Comment (oliverochman) [inline public/manifest.json]: remove the "icons" key here

File: public/robots.txt [hunk 2]
```diff
@@ -0,0 +1,3 @@
+# https://www.robotstxt.org/robotstxt.html
```
Comment (oliverochman) [inline public/robots.txt]: remove this file

File: src/index.css [hunk 3]
```diff
@@ -0,0 +1,13 @@
+body {
```
Comment (oliverochman) [inline src/index.css]: remove this file

File: src/index.js [hunk 4]
```diff
@@ -0,0 +1,17 @@
...
+import * as serviceWorker from './serviceWorker';
+
+ReactDOM.render(
+  <React.StrictMode>
+    <App />
+  </React.StrictMode>,
+  document.getElementById('root')
+);
+
+// If you want your app to work offline and load faster, you can change
```
Comment (oliverochman) [inline src/index.js]: remove comments

**Review by oliverochman — CHANGES_REQUESTED** [review]
please remove the "public/favicon.ico" file

**Review by emtalen — APPROVED** [review]

**Pull request merged**